src/serac/physics/thermal_conduction.hpp

```diff
@@ -70,6 +70,13 @@ class ThermalConduction : public BasePhysics {
     double cp;
     double rho;
 
+    // Nonlinear reaction information
+    std::optional<std::function<double(double)>> reaction_func;
+    std::optional<std::function<double(double)>> d_reaction_func;
```

Comment:
`std::function` is already nullable (it has an `operator bool` etc) - I think we can leave off the `std::optional` here

Reply:
TIL!